litmus-portal/frontend/src/views/ChaosWorkflows/BrowseCluster/index.tsx

```diff
@@ -346,16 +336,13 @@ const BrowseCluster = () => {
                       paginationData.rowsPerPage
                   )
                   .map((data: Cluster) => (
-                    <TableRow
-                      data-cy="browseScheduleData"
-                      key={data.cluster_id}
-                    >
+                    <TableRow data-cy="browseData" key={data.cluster_id}>
                       <TableData data={data} />
                     </TableRow>
                   ))
               ) : (
                 <TableRow>
-                  <TableCell data-cy="browseScheduleNoData" colSpan={0}>
+                  <TableCell data-cy="browseClusterData" colSpan={0}>
```

Comment:
Change it to browseClusterNoData...as it will be rendered when there is no data.